server/sonar-server/src/test/resources/org/sonar/server/db/migrations/v51/RenameComponentRelatedParamsInIssueFiltersMigrationTest/execute.xml

```diff
@@ -16,7 +16,7 @@
       user_login="michael"
       shared="[false]"
       description="to be updated"
-      data="statuses=OPEN|sort=SEVERITY|asc=true|projects=ABC"
+      data="statuses=OPEN|sort=SEVERITY|asc=true|projectUuids=ABC"
       created_at="2013-06-10"
       updated_at="2013-06-10" />
 
```

Comment:
Can please add a comment above each entry in order to explain what should be updated ? It will help to understand what is tested.